source/agora/consensus/data/UTXO.d

```diff
@@ -126,7 +122,7 @@ public class TestUTXOSet
     }
 
     /// Get an UTXO, does not return double spend
-    private bool findUTXO_ (Hash utxo, out UTXO value)
+    public bool findUTXO (Hash utxo, out UTXO value)
```

Comment:
Is there any reason why the access modifier changed to `public`? The code means that we can call it directly.

Reply:
Yes, that was the point of this change. Before `findUTXO` would alias to `peekUTXO`, meaning no double spend protection. This was done to keep tests from passing, because the `TestUTXOSet` didn't originally have double spend protection, so adding it would have meant fixing tons of tests otherwise. Which @kchulj eventually did :)

Reply:
Yes, that allows us to call it directly.
In most unittests, the parameter is a `delegate` so I assumed that passing the `delegate` is better.
Technically, `findUTXO` and `getUTXOFinder` should do the same thing after this change.